src/utils/Theme/blocks/ComponentBlocks.tsx

```diff
@@ -1,47 +1,71 @@
 import React from 'react';
 import { ReablocksTheme } from '@/utils/Theme/themes';
 import { JsonTree } from '@/layout/Tree';
+import { Button } from '@/elements';
+import { Notifications, NotificationsContext } from '@/layers';
 
 export const ComponentBlocks = ({
   components
 }: {
   components: ReablocksTheme['components'];
-}) => {
-  return (
-    <div
-      style={{
-        padding: '4px 8px',
-        color: 'var(--body-color)',
-        width: '100%'
-      }}
-    >
-      {components ? (
-        <>
-          {Object.keys(components).map(key => (
-            <div key={key} className="pb-3">
-              <h3
-                style={{
-                  fontSize: '24px',
-                  fontWeight: 500,
-                  marginRight: 50,
-                  marginBottom: '4px',
-                  maxWidth: 300,
-                  textTransform: 'capitalize'
-                }}
-              >
-                {key}
-              </h3>
-              <JsonTree
-                className="border border-gray-700 rounded p-3"
-                expandDepth={Infinity}
-                data={components[key]}
-              />
-            </div>
-          ))}
-        </>
-      ) : (
-        <p>⚠️ No components(s) defined</p>
+}) => (
+  <Notifications>
```

Comment:
i probably would have skipped this but lgtm

Reply:
I think better with notification to see action was success